Makefile

```diff
@@ -451,6 +453,7 @@ install-program: installdirs $(BIN_PROGRAMS) $(PKGLIBEXEC_PROGRAMS)
 	@$(NORMAL_INSTALL)
 	$(INSTALL_PROGRAM) $(BIN_PROGRAMS) $(DESTDIR)$(bindir)
 	$(INSTALL_PROGRAM) $(PKGLIBEXEC_PROGRAMS) $(DESTDIR)$(pkglibexecdir)
+	$(INSTALL_PROGRAM) plugins/lplugin_* $(DESTDIR)$(plugindir)
```

Comment:
This is installing files that are not specified in the Makefile rule as dependency of this rule.

Reply:
If they're python, they don't have deps.  But yes, we should create a variable for these!